### ISSUE TITLE
Questions about retraining the deepseg_gm model and data preprocessing

### ISSUE THREAD

**Issue by anacis** [spinalcordtoolbox/spinalcordtoolbox#3088]
Dear SCT Team,

I am trying to retrain the deepseg_gm model on PSIR contrast images (this is a contrast that we are developing in our lab) since when we initially tested the current sct model on our contrast, the results very suboptimal.

I had some questions about how you preprocessed the data for this model:

Augmentation:
How exactly was the data augmented - did you run each type of augmentation on every slice with the specified probabilities? Do we have to do this for their labels as well? Is one augmentation applied on top of other augmentations or do you make multiple copies of the original data.

Channel Shifting: my MRI images are grayscale so how exactly does channel shifting work for single-channel images? 

Normalization: The paper says to do mean centering and standard deviation normalization of the volume intensities. Do we do that for each slice or for the full volume - since we are inputting 2d slices into model?

Thank you so much for all your help!
Ana Cismaru
UCSF Henry Lab

**Comment by jcohenadad** [comment]
Hi @anacis, thank you for reaching out!

All our models are now trained within the [ivadomed framework](https://ivadomed.org/en/latest/), and can be used with the new general SCT function `sct_deepseg`. We are happy to assist you setting up the training configuration for your PSIR data. As a start, we suggest you go through the [first tutorial](https://ivadomed.org/en/latest/tutorials/one_class_segmentation_2d_unet.html). If you have any issue, please feel free to post directly on the [GitHub issue tracking of ivadomed](https://github.com/ivadomed/ivadomed/issues), we will be happy to assist you 😊 .

**Comment by joshuacwnewton** [comment]
Closing this issue, as it has been >2 years with no response, and this issue does not pertain to SCT bugs/feature requests/etc.

As a follow-up, though, I will create an issue in `neuropoly/idea-projects` for https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/3046, which will cover the task of retraining `deepseg_gm` and others using PyTorch. Any further activity for retraining `deepseg_gm` can be discussed there. :)
